e2e/cypress/tests/integration/account_settings/profile/email_spec.ts

```diff
@@ -191,7 +207,7 @@ describe('Profile > Profile Settings > Email', () => {
         // # Wait for one second for the mail to be sent out.
         cy.wait(TIMEOUTS.ONE_SEC);
 
-        cy.getRecentEmail({username, email}).then((data) => {
+        cy.getRecentEmail(testUser).then((data) => {
```

Comment:
Tests here failed on me, and got passing when reverted both into `cy.getRecentEmail({username, email})...`